direct/nn/vsharp/vsharp.py

```diff
@@ -271,13 +272,14 @@ def forward(
         masked_kspace: torch.Tensor
             Masked k-space of shape (N, coil, height, width, complex=2).
         sensitivity_map: torch.Tensor
-            Sensitivity map of shape (N, coil, height, width, complex=2). Default: None.
+            Sensitivity map of shape (N, coil, height, width, complex=2).
         sampling_mask: torch.Tensor
+            Sampling mask of shape (N, 1, height, width, 1).
 
         Returns
         -------
-        image: torch.Tensor
-            Output image of shape (N, height, width, complex=2).
+        out : list of torch.Tensors
+            List of output images of shape (N, height, width, complex=2).
```

Comment:
Wouldn't it work if you do `::math::` or so?

Reply:
You mean on this `(N, height, width, complex=2)`?

Reply:
It looks good without ::math:: as it turns out